### PR TITLE
[FLINK-13485] [chinese-translation] Translate "Table API Example Walkthrough" page into Chinese

### PR DESCRIPTION
**Pull request by AT-Fieldless**
## What is the purpose of the change

Translate "Table API Example Walkthrough" page into Chinese.

## Brief change log

- translate "Table API Example Walkthrough" page into Chinese.

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): (no)
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: (no)
  - The serializers: (no)
  - The runtime per-record code paths (performance sensitive): (no)
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Yarn/Mesos, ZooKeeper: (no)
  - The S3 file system connector: (no)

## Documentation

  - Does this pull request introduce a new feature? (no)

### CODE REVIEW

File: docs/getting-started/walkthroughs/table_api.zh.md [hunk 1]
```diff
@@ -300,10 +303,11 @@ This query consumes all records from the `transactions` table, calculates the re
 {% endhighlight %}
 
 ## Adding Windows
+## 添加窗口
```
Comment (xintongsong) [inline docs/getting-started/walkthroughs/table_api.zh.md]: 英文标题没有删掉

File: docs/getting-started/walkthroughs/table_api.zh.md [hunk 2]
```diff
@@ -380,10 +384,10 @@ val tEnv = StreamTableEnvironment.create(env)
 </div>
 </div>
 
-The second step is to migrate from a bounded data source to an infinite data source.
-The project comes with an `UnboundedTransactionTableSource` that continuously creates transaction events in real-time.
-Similar to the `BoundedTransactionTableSource` this table is backed by data generated in memory to avoid any dependencies on external systems.
-In practice, this table might read from a streaming source such as Apache Kafka, AWS Kinesis, or Pravega.
+第二步就是把有界的数据源替换成无限的数据源。
+这个项目通过 `UnboundedTransactionTableSource` 来实时生成交易事件。
```
Comment (xintongsong) [inline docs/getting-started/walkthroughs/table_api.zh.md]: 译文没包含continuously

File: docs/getting-started/walkthroughs/table_api.zh.md [hunk 3]
```diff
@@ -399,27 +403,29 @@ tEnv.registerTableSource("transactions", new UnboundedTransactionTableSource)
...
 </div>
 
-And that's it, a fully functional, stateful, distributed streaming application!
-The query continuously consumes the stream of transactions, computes the hourly spendings, and emits results as soon as they are ready.
-Since the input is unbounded, the query keeps running until it is manually stopped.
-And because the Job uses time window-based aggregations, Flink can perform specific optimizations such as state clean up when the framework knows that no more records will arrive for a particular window.
+这就是一个功能齐全、有状态的分布式流式应用！
+这个查询会持续处理交易流，计算每小时的消费额，然后实时输出结果。
+由于输入是无界的，因此查询将一直运行，直到手动停止为止。
+因为这个作业使用了基于时间的聚合，Flink 可以使用一些特定的优化，比如当系统知道一个特定的窗口不会再有新的数据到来，它就会对状态进行清理。
```
Comment (xintongsong) [inline docs/getting-started/walkthroughs/table_api.zh.md]: “基于时间的聚合”->“基于时间窗口的聚合”：这点很重要，如果不是基于窗口，flink无法进行此类优化

**Comment by AT-Fieldless** [comment]
@xintongsong Thanks for your review, and I update my commit follow your advice.I delete some redundant blank lines but maybe still some remains.Very grateful if you find them and I will delete them.

**Comment by xintongsong** [comment]
Thanks for addressing the comments, @AT-Fieldless. I'll take a look asap.

FYI, you can build the docs locally with `cd flink/doc && sh build_docs.sh -p -i` and check the style with your browser at `http://0.0.0.0:4000`.

File: docs/getting-started/walkthroughs/table_api.zh.md [hunk 4]
```diff
@@ -158,12 +157,12 @@ val tEnv = BatchTableEnvironment.create(env)
...
+#### 注册表
 
-Next, tables are registered in the execution environment that you can use to connect to external systems for reading and writing both batch and streaming data.
-A table source provides access to data stored in external systems; such as a database, a key-value store, a message queue, or a file system.
-A table sink emits a table to an external storage system.
-Depending on the type of source and sink, they support different formats such as CSV, JSON, Avro, or Parquet.
+接下来，表将会被注册到运行环境之中，这样你就可以用它们连接外部系统以读取或写入批数据或流数据。
+表数据源提供对存储在外部系统中的数据的访问；例如数据库、键-值存储、消息队列或文件系统。
+表接收器则将表中的数据发送到外部存储系统。
+根据 source 或 sink 的类型，它们支持不同的格式，如 CSV、JSON、Avro 或 Parquet。
```
Comment (xintongsong) [inline docs/getting-started/walkthroughs/table_api.zh.md]: 这一段中的表述不统一。
source/sink vs. 数据源/接收器

**Comment by xintongsong** [comment]
@AT-Fieldless 
I've reviewed the updates. Left comment on one minor issue. The rest of the PR looks good to me.

**Comment by AT-Fieldless** [comment]
@xintongsong I change '数据源' and '接收器' with source and sink here, but there remain some '数据源' in other paragraphs, should I change them?

**Comment by xintongsong** [comment]
I would suggest we differentiate the occurrences of 'source' into two categories and handle them differently.
- For 'source' that refers to the source operator, or the source vertex of the job graph, we treat it as a proper noun and do not translate it.
- For 'source' that refers to where the data comes from in general, we translate it to '数据源'.

Above are my personal opinions. Discussions are welcomed if you have different opinions.

**Comment by AT-Fieldless** [comment]
Ok, I agree with you and I hava not problem

File: docs/getting-started/walkthroughs/table_api.zh.md [hunk 5]
```diff
@@ -299,11 +297,11 @@ This query consumes all records from the `transactions` table, calculates the re
...
 
-## Adding Windows
+## 添加窗口
 
-Grouping data based on time is a typical operation in data processing, especially when working with infinite streams.
-A grouping based on time is called a [window]({{ site.baseurl }} /dev/stream/operators/windows.html) and Flink offers flexible windowing semantics.
-The most basic type of window is called a `Tumble` window, which has a fixed size and whose buckets do not overlap.
+根据时间进行分组在数据处理中是一种很常见的方式，特别是在处理无限的数据流时。
+基于时间的分组称为[窗口]({{ site.baseurl }}/zh/dev/stream/operators/windows.html) ，Flink 提供了灵活的窗口语义。
+其中最基础的是 `滚动窗口`，它具有固定大小且窗口之间不重叠。
```
Comment (wuchong) [inline docs/getting-started/walkthroughs/table_api.zh.md]:
```suggestion
其中最基础的是 `Tumble` window （滚动窗口），它具有固定大小且窗口之间不重叠。
```

Tumble  这里是更多的是指 代码中的 `Tumble`，所以不翻译。

**Pull request closed without merging**